symphony/bdk/core/config/model/bdk_certificate_config.py

```diff
@@ -21,3 +21,20 @@ def is_valid(self) -> bool:
         :return: true if the certificate configuration is valid.
         """
         return not (self.path is not None and self.content != "")
+
+    def setContent(self, certificate_content):
+        """Sets ceritficate content and overrides path to None to keep certificate config valid
+        Args:
```

Comment:
Please use restructured text format. You can configure it on PyCharm for instance: https://www.jetbrains.com/help/pycharm/settings-tools-python-integrated-tools.html -> docstring format.

Reply:
typo in certificate too